TryAtSoftware.Extensions.Collections.Tests/TryAtSoftware.Extensions.Collections.Tests.csproj

```diff
@@ -25,6 +25,7 @@
 
     <ItemGroup>
       <ProjectReference Include="..\TryAtSoftware.Extensions.Collections\TryAtSoftware.Extensions.Collections.csproj" />
+      <ProjectReference Include="..\TryAtSoftware.Extensions.Reflection.Tests\TryAtSoftware.Extensions.Reflection.Tests.csproj" />
```

Comment:
We do not need to cross-reference test projects.

Reply:
In order to get the random number, it has been done.

Reply:
Use `TryAtSoftware.Randomizer` instead.

Reply:
I am unable to figure out, how to use `Randomizer`, requesting for your help
Meanwhile, I have removed cross-referencing and commented the first test function.

Reply:
You can check out this PR for a better understanding: https://github.com/TryAtSoftware/Extensions/pull/66

Reply:
Thanks for it. I have implmented the code changes accordingly.

Reply:
Hey, any update?